### PR TITLE
Added a benchmark script

### PR DESCRIPTION
**Pull request by vporton**

### CODE REVIEW

File: benchmark.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const web3 = new Web3('https://data-seed-prebsc-1-s1.binance.org:8545/');
+
+async function main() {
+    const numberOfRuns = process.argv[2] ? process.argv[2] : 10;
+
+    const tx = {
+        from: addressFrom,
+        to: addressTo,
+        value: web3.utils.toWei('100', 'ether'),
+        gas: web3.utils.toHex('21000'),
```
Comment (DmytroNazarenko) [inline benchmark.js:16]: Mb read gas as script argument, so we can run benchmark for different gas values? And by default it will be 21000

Comment (vporton) [reply benchmark.js:16]: You can do this, but I myself is now busy with another project.

File: benchmark.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const Web3 = require('web3');
+const mineGasForTransaction = require('./skale-miner').mineGasForTransaction
+
+// Variables definition
+const addressFrom = '0x6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b';
+const addressTo = '0xB90168C8CBcd351D069ffFdA7B71cd846924d551';
+const web3 = new Web3('https://data-seed-prebsc-1-s1.binance.org:8545/');
```
Comment (DmytroNazarenko) [inline benchmark.js:7]: Is this some specific endpoint or may we replace web3 by web3-utils for `toWei` and `toHex` functions?

Comment (vporton) [reply benchmark.js:7]: These are randomly generated addresses, just for testing.

**Pull request merged**